### PR TITLE
i18n(fr): translate guides/deploy

### PR DESCRIPTION
**Pull request by QuiiBz**
#### What kind of changes does this PR include?

- Translated content

#### Description

Translate `guides/deploy.md` page to french.

### CODE REVIEW

File: src/pages/fr/guides/deploy.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Déployer votre Site Astro
+description: Comment déployer votre site Astro sur le web.
+layout: ~/layouts/MainLayout.astro
+setup: import DeployGuidesNav from '~/components/DeployGuidesNav.astro';
+i18nReady: true
+---
+**Prêt à construire et déployer votre site Astro ?** Suivez un de nos guides sur différents services de déploiment, ou scrollez plus bas pour avoir des orientations générales sur comment déployer un site Astro.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md:8]: Typo
```suggestion
**Prêt à construire et déployer votre site Astro ?** Suivez un de nos guides sur différents services de déploiement, ou scrollez plus bas pour avoir des orientations générales sur comment déployer un site Astro.
```

Comment (QuiiBz) [reply src/pages/fr/guides/deploy.md:8]: Thanks, I think I copy/pasted this typo everywhere 😅

File: src/pages/fr/guides/deploy.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+<DeployGuidesNav />
+
+## Options de déploiment rapide
+
+Vous pouvez construire et déployer rapidement un site Astro via de nombreux hébergeurs, via l'interface de leur site internet ou via une CLI.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md:14]: Trying to simplify the sentence, what do you think about this?
```suggestion
Vous pouvez construire et déployer rapidement un site Astro vers de nombreux hébergeurs, via l'interface de leur site internet ou une CLI.
```

File: src/pages/fr/guides/deploy.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+### Interface d'un hébergeur
+
+Une façon rapide de déployer votre site internet est de connecter le repository Git (ex. GitHub, GitLab, Bitbucket) de votre projet Astro à un hébergeur, afin de profiter du déploiment continu en utilisant Git.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md]: Translation and typo
```suggestion
Une façon rapide de déployer votre site internet est de connecter le dépôt Git (ex. GitHub, GitLab, Bitbucket) de votre projet Astro à un hébergeur, afin de profiter du déploiement continu en utilisant Git.
```

Comment (QuiiBz) [reply src/pages/fr/guides/deploy.md]: Should we use "dépôt" instead of "repository" everywhere?

File: src/pages/fr/guides/deploy.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+Ces hébergeurs détectent automatiquement les push sur le repository de votre projet Astro, build votre site et le déploient sur le web sur une URL personnalisée ou sur votre nom de domaine personnel. La plupart du temps, mettre en place le déploiment sur ces platformes suivra les étapes suivantes :
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md]: Typo and translation
```suggestion
Ces hébergeurs détectent automatiquement les push sur le dépôt de votre projet Astro, build votre site et le déploient sur le web sur une URL personnalisée ou sur votre nom de domaine personnel. La plupart du temps, mettre en place le déploiement sur ces plateformes suivra les étapes suivantes :
```

Comment (QuiiBz) [reply src/pages/fr/guides/deploy.md]: Replaced every occurence of "repository" to "depôt"

File: src/pages/fr/guides/deploy.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+1. Ajoutez votre repository à un provider Git (ex. sur GitHub, GitLab, Bitbucket)
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md]: Translation. Also we can translate 'provider' but I don't have anything good right now
```suggestion
1. Ajoutez votre dépôt à un provider Git (ex. sur GitHub, GitLab, Bitbucket)
```

File: src/pages/fr/guides/deploy.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+1. Choisissez un hébergeur qui supporte le *déploiment continu* (ex. [Netlify](/en/guides/deploy/netlify/) ou [Vercel](/en/guides/deploy/vercel/)) et importez votre repository Git en tant que nouveau projet / site.
+
+    De nombreux hébergeurs vont reconnaitre votre projet en tant que site Astro, et devraient automatiquement appliquer la configuration appropriée pour construire et déployer votre site comme montré ci-dessous. (Dans le cas contraire, ces paramètres peuvent être modifiés.)
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md:26]: Little missing 'accent' 
```suggestion
    De nombreux hébergeurs vont reconnaître votre projet en tant que site Astro, et devraient automatiquement appliquer la configuration appropriée pour construire et déployer votre site comme montré ci-dessous. (Dans le cas contraire, ces paramètres peuvent être modifiés.)
```

File: src/pages/fr/guides/deploy.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+    - **Commande de build:** `astro build` ou `npm run build`
+    - **Dossier de publication:** `dist`
+    :::
+
+1. Cliquez sur "Déployer" et votre nouveau site internet sera créé avec une URL unique (ex. `new-astro-site.netlify.app`).
+
+
+L'hébergeur va automatiquement inspecter votre branche main de votre dépôt, et dès que des changements seront détectés, va reconstruire et republier votre site. Ces paramètres peuvent typiquement être modifiés dans l'interface de votre hébergeur.
+
+### déploiement via une CLI
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md:38]: Capitalized title
```suggestion
### Déploiement via une CLI
```

File: src/pages/fr/guides/deploy.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+Certains hébergeurs proposent leur propre interface de ligne de commande (CLI) que vous pouvez installer globalement sur votre machine en utilisant npm. Souvent, utiliser une CLI pour déployer ressemble à ceci :
+
+1. Installez la CLI de votre hébergeur, par example :
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md:42]: Nit & typo
```suggestion
1. Installez globalement la CLI de votre hébergeur, par exemple :
```

File: src/pages/fr/guides/deploy.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+    ```bash
+    npm install --global netlify-cli
+    ```
+
+1. Executez la CLI et suivez les instructions de connexion, mise en place etc.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md:48]: Nit
```suggestion
1. Exécutez la CLI et suivez les instructions de connexion, mise en place etc.
```

File: src/pages/fr/guides/deploy.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+```bash
+npm run build
+```
+
+Par défaut, le dossier de build sera placé à `dist/`. Ce dossier peut être changé en utilisant [l'option `outDir`](/fr/reference/configuration-reference/#outdir).
+
+## Ajouter un Adapteur pour du SSR
+
+:::note
+Avant de déployer votre site Astro avec du [SSR (server-side rendering)](/fr/guides/server-side-rendering/) activé, vérifiez que vous avez :
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/deploy.md:79]: Ici on peut traduire "server-side rendering" en "rendu côté serveur" tu trouves pas?

Comment (QuiiBz) [reply src/pages/fr/guides/deploy.md:79]: Sur ces termes techniques je suis pas sûr que ça soit plus parlant pour l'utilisateur. Il faudrait regarder sur les autres pages en FR/autres si c'est traduit ou non

File: src/pages/fr/guides/deploy.md (hunk 3 above)
Comment (MoustaphaDev) [reply src/pages/fr/guides/deploy.md]: Sorry, didn't see your replies pop in. Yeah I think so

File: src/pages/fr/guides/deploy.md (hunk 10 above)
Comment (MoustaphaDev) [reply src/pages/fr/guides/deploy.md:79]: Je suis d'accord, j'ai check des pages de la doc et effectivement c'est pas traduit dans ce contexte

**Review by MoustaphaDev — APPROVED** [review]
Looks good!

File: src/pages/fr/guides/deploy.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+### Interface d'un hébergeur
+
+Une façon rapide de déployer votre site internet est de connecter le dépôt Git (ex. GitHub, GitLab, Bitbucket) de votre projet Astro à un hébergeur, afin de profiter du déploiement continu en utilisant Git.
+
+Ces hébergeurs détectent automatiquement les push sur le dépôt de votre projet Astro, build votre site et le déploient sur le web sur une URL personnalisée ou sur votre nom de domaine personnel. La plupart du temps, mettre en place le déploiement sur ces plateformes suivra les étapes suivantes :
+
+1. Ajoutez votre dépôt à un provider Git (ex. sur GitHub, GitLab, Bitbucket)
+
+1. Choisissez un hébergeur qui supporte le *déploiement continu* (ex. [Netlify](/en/guides/deploy/netlify/) ou [Vercel](/en/guides/deploy/vercel/)) et importez votre dépôt Git en tant que nouveau projet / site.
```
Comment (delucis) [inline src/pages/fr/guides/deploy.md:24]: We always use the `/fr/` links to make sure we don’t pull a reader over to the English docs site.

```suggestion
1. Choisissez un hébergeur qui supporte le *déploiement continu* (ex. [Netlify](/fr/guides/deploy/netlify/) ou [Vercel](/fr/guides/deploy/vercel/)) et importez votre dépôt Git en tant que nouveau projet / site.
```

File: src/pages/fr/guides/deploy.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+Par défaut, le dossier de build sera placé à `dist/`. Ce dossier peut être changé en utilisant [l'option `outDir`](/fr/reference/configuration-reference/#outdir).
+
+## Ajouter un Adapteur pour du SSR
+
+:::note
+Avant de déployer votre site Astro avec du [SSR (server-side rendering)](/fr/guides/server-side-rendering/) activé, vérifiez que vous avez :
+
+- Installé [l'adapteur approprié](/fr/guides/server-side-rendering/#activation-du-mode-ssr-dans-votre-projet) en tant que dépendance de votre projet (que ça soit manuellement, ou en utilisant la commande de l'adapteur `astro add`, ex. `npx astro add netlify`).
+- [Ajouté l'adapteur](/fr/reference/configuration-reference/#integrations) à votre fichier `astro.config.mjs` si vous installez manuellement. (La commande `astro add` fera cette étape automatiquement !)
```
Comment (delucis) [inline src/pages/fr/guides/deploy.md:82]: Need to fix this `#` link for the French translation:

```suggestion
- [Ajouté l'adapteur](/fr/reference/configuration-reference/#intégrations) à votre fichier `astro.config.mjs` si vous installez manuellement. (La commande `astro add` fera cette étape automatiquement !)
```

**Review by delucis — CHANGES_REQUESTED** [review]
Awesome work @QuiiBz! And thanks for reviewing @MoustaphaDev.

Just a couple of links here that need fixing:

**Comment by QuiiBz** [comment]
Thanks for the review @delucis and @MoustaphaDev. Everything should be fixed

**Review by delucis — APPROVED** [review]

**Review by thomasbnt — APPROVED** [review]

**Pull request merged**